build.gradle

```diff
@@ -16,9 +16,13 @@ import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
 buildscript {
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-SNAPSHOT")
-        // 1.2.0 -> 1.2.0.0, and 1.2.0-SNAPSHOT -> 1.2.0.0-SNAPSHOT
-        opensearch_build = opensearch_version.replaceAll(/(\.\d)([^\d]*)$/, '$1.0$2')
+        opensearch_version = System.getProperty("opensearch.version", "2.0.0-alpha1-SNAPSHOT")
```

Comment:
Infra team fixed this part, you can refer to this PR https://github.com/opensearch-project/ml-commons/pull/262/files

Reply:
Changed it to be the same as ml-commons

Reply:
Thanks, btw, make sure the built AD artifact name is `opensearch-anomaly-detection-2.0.0.0-alpha1-SNAPSHOT`

Reply:
It actually gives a zip of `opensearch-anomaly-detection-2.0.0.0-SNAPSHOT`. Same case for ml-commons built zip (just tried on latest code)

Reply:
oh,I guess infra team's fix break this part for `ml-commons`, will take a look

Reply:
This one can generate artifact with alpha1, `./gradlew clean;./gradlew build -Dbuild.version_qualifier=alpha1`. But artifact generated by `./gradlew build` has no `alpha1` in name even we set `opensearch_version = System.getProperty("opensearch.version", "2.0.0-alpha1-SNAPSHOT")`. @peterzhuamazon  is this what we expected?